src/Scene/NodeProcess.js

```diff
@@ -479,16 +479,19 @@ NodeProcess.prototype.processNode = function processNode(node, camera, params) {
  * @returns {NodeProcess_L7.NodeProcess.prototype.frustumCullingOBB.node@pro;camera@call;getFrustum@call;intersectsBox}
  */
 
-var quaternion = new THREE.Quaternion();
+const frustum = new THREE.Frustum();
+const obbMatrixWorld = new THREE.Matrix4();
 
 NodeProcess.prototype.frustumCullingOBB = function frustumCullingOBB(node, camera) {
-    // position in local space
-    var position = node.OBB().worldToLocal(camera.position().clone());
-    position.z -= node.distance;
+    // Move camera in OBB local space
+    obbMatrixWorld.copy(camera.camera3D.matrixWorldInverse);
```

Comment:
Three does it automatically (see matrixAutoUpdate attribute). However, in the current version, we can't add other objects to the node's children without breaking the quadtree processes.